apps/api/src/routes/account.routes.ts

```diff
@@ -458,98 +458,5 @@ export function configureAccountRoutes(
-   *                 trades:
-   *                   type: array
-   *                   items:
-   *                     type: object
-   *                     properties:
-   *                       id:
-   *                         type: string
-   *                         description: Unique trade ID
-   *                       teamId:
-   *                         type: string
-   *                         description: Team ID that executed the trade
-   *                       competitionId:
-   *                         type: string
-   *                         description: ID of the competition this trade is part of
-   *                       fromToken:
-   *                         type: string
-   *                         description: Token address that was sold
-   *                       toToken:
-   *                         type: string
-   *                         description: Token address that was bought
-   *                       fromAmount:
-   *                         type: number
-   *                         description: Amount of fromToken that was sold
-   *                       toAmount:
-   *                         type: number
-   *                         description: Amount of toToken that was received
-   *                       price:
-   *                         type: number
-   *                         description: Price at which the trade was executed
-   *                       success:
-   *                         type: boolean
-   *                         description: Whether the trade was successfully completed
-   *                       error:
-   *                         type: string
-   *                         description: Error message if the trade failed
-   *                       reason:
-   *                         type: string
-   *                         description: Reason provided for executing the trade
-   *                       timestamp:
-   *                         type: string
-   *                         format: date-time
-   *                         description: Timestamp of when the trade was executed
-   *                       fromChain:
-   *                         type: string
-   *                         description: Blockchain type of the source token
-   *                       toChain:
-   *                         type: string
-   *                         description: Blockchain type of the destination token
-   *                       fromSpecificChain:
-   *                         type: string
-   *                         description: Specific chain for the source token
-   *                       toSpecificChain:
-   *                         type: string
-   *                         description: Specific chain for the destination token
-   *       401:
-   *         description: Unauthorized - Missing or invalid authentication
-   *       500:
-   *         description: Server error
-   */
-  router.get("/trades", controller.getTrades);
```

Comment:
this is a duplicate of line 392

Reply:
Looks like the openapi comment is very different for each duplicate, so just to be sure you're choosing the correct version of the comment...

Reply:
Good call, I believe the one I chose is correct.